cli-plugins/plugin/plugin.go

```diff
@@ -158,7 +158,7 @@ func newPluginCommand(dockerCli *command.DockerCli, plugin *cobra.Command, meta
 		CompletionOptions: cobra.CompletionOptions{
 			DisableDefaultCmd:   false,
 			HiddenDefaultCmd:    true,
-			DisableDescriptions: true,
+			DisableDescriptions: os.Getenv("DOCKER_CLI_DISABLE_COMPLETION_DESCRIPTION") != "",
```

Comment:
cc @crazy-max I think descriptions are enabled by default in buildx; do you think it'd make sense for it to  also support  this env-var? (at least when running as plugin)

cc @glours (not sure what currently the default  is on compose, but "same" question for there)